docs/concepts/command-service.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Command Service
+headline: Command Service
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Command Service</h2> 
+
+Commands service concept to be added.
+
+
```

Comment:
Please also do not add empty lines. They will add up to the difference in the next update.